packages/aws/data_stream/lambda_logs/elasticsearch/ingest_pipeline/aws-lambda-json.yml

```
@@ -0,0 +1,498 @@
+---
+description: Pipeline for parsing AWS lambda logs in JSON format
+processors:
+- json:
+    field: event.original
+    target_field: parsed
+    ignore_failure: true
+
+# 1. EMF metrics (_aws.Timestamp)
+- date:
+    if: "ctx.parsed?._aws != null"
+    field: parsed._aws.Timestamp
+    target_field: "@timestamp"
+    formats: ["UNIX_MS"]
+    ignore_failure: true
+
+# 2. Powertools structured logs (timestamp field)
+- date:
+    if: "ctx.parsed?.timestamp != null"
+    field: parsed.timestamp
+    target_field: "@timestamp"
+    formats: ["yyyy-MM-dd HH:mm:ss,SSSZ"]
+    ignore_failure: true
+
+# 3. Platform report logs (time field)
+- date:
+    if: "ctx.parsed?.time != null"
+    field: parsed.time
+    target_field: "@timestamp"
+    formats: ["ISO8601"]
+    ignore_failure: true
+
+- date:
+    if: "ctx.parsed?._aws?.Timestamp != null"
```

This processor appears to be doing the same as what's done in the first date processor. Why are we handling it separately here instead of adding a condition in the original check?
```suggestion
    if: ctx.parsed?._aws != null && ctx.parsed?._aws?.Timestamp != null
```